### PR TITLE
cetz:0.0.1

### PR DESCRIPTION
**Pull request by johannes-wolf**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: CeTZ (formerly `typst-canvas`) is a package providing a canvas environment for drawing with Typst. Its API is heavily inspired by TikZ and Processing.

URL: https://github.com/johannes-wolf/typst-canvas

### CODE REVIEW

**Comment by laurmaedje** [comment]
I understand the desire for versioning things together, but at the same time it seems like a big waste to download a big package bundle from which only a few KB are actually used, especially in the web app.

**Comment by laurmaedje** [comment]
Do you also value these files being part of the tar.gz bundle? If not, we could consider adding an `exclude` key to the manifest with globs of files that should not be part of the bundle.

**Comment by johannes-wolf** [comment]
> Do you also value these files being part of the tar.gz bundle? If not, we could consider adding an `exclude` key to the manifest with globs of files that should not be part of the bundle.

No, they are only useful for the readme. An `exclude` key would be perfect.

**Comment by laurmaedje** [comment]
I implemented an `exclude` key that takes an array of globs.

**Comment by johannes-wolf** [comment]
Thank you, I have added all the images and the pdf to the `exclude` list.
This PR is ready for review.

File: packages/preview/cetz-0.0.1/typst.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+version = "0.0.1"
+repository = "https://github.com/johannes-wolf/typst-canvas"
+entrypoint = "lib.typ"
+authors = [
+  "Johannes Wolf <https://github.com/johannes-wolf>",
+  "fenjalien <https://github.com/fenjalien>"
+]
+license = "Apache-2.0"
+description = "Drawing with Typst made easy, providing an API inspired by TikZ and Processing."
+exclude = [ "*.png", "manual.pdf", "manual.typ" ]
```
Comment (laurmaedje) [inline packages/preview/cetz-0.0.1/typst.toml:12]: is it intentional that `gallery` isn't excluded completely? the three Typst files aren't big, but maybe a bit pointless.

Comment (johannes-wolf) [reply packages/preview/cetz-0.0.1/typst.toml]: Done.

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**